spec/system/topic_bulk_select_spec.rb

```diff
@@ -24,6 +24,29 @@ def open_bulk_actions_modal(topics_to_select = nil, action)
     expect(topic_bulk_actions_modal).to be_open
   end
 
+  context "when dismissing topics" do
+    fab!(:topic) { Fabricate(:topic, user: admin) }
+    fab!(:post1) { create_post(user: admin, topic: topic) }
+    fab!(:post2) { create_post(topic: topic) }
+
+    it "removes the topics from the list" do
+      sign_in(admin)
+      visit("/unread")
```

Comment:
Can we add a test for `visit("/new")` as well?

Reply:
Absolutely we should! 👍